misc/xml-to-record-converter/src/test/resources/ballerina/sample_9.bal

```diff
@@ -1,43 +1,37 @@
-type Ns1\:element1 record {
+type Ns1_Element1 record {
     @xmldata:Attribute
     string ns1\:attribute1;
     @xmldata:Attribute
     string ns2\:attribute2;
 };
 
-@xmldata:Namespace {
-    prefix: "xsi",
-    uri: "http://www.w3.org/2001/XMLSchema-instance"
-}
-type Ns2\:null record {
+type Ns2_Null record {
+    @xmldata:Attribute
+    string xmlns\:xsi = "http://www.w3.org/2001/XMLSchema-instance";
     @xmldata:Attribute
     string xsi\:nil;
 };
 
-type Ns2\:array record {
+type Ns2_Array record {
     string[] ns2\:item;
 };
 
-type Ns2\:element2 record {
+type Ns2_Element2 record {
     string ns2\:subelement;
     decimal ns2\:number;
     boolean ns2\:boolean;
-    Ns2\:null ns2\:null;
-    Ns2\:array ns2\:array;
+    Ns2_Null ns2\:null;
+    Ns2_Array ns2\:array;
 };
 
 @xmldata:Name {
     value: "root"
 }
-@xmldata:Namespace {
-    prefix: "ns1",
-    uri: "http://namespace1.com"
-}
-@xmldata:Namespace {
-    prefix: "ns2",
-    uri: "http://namespace2.com"
-}
 type Root record {
```

Comment:
Do we have a test case for default namespaces like below

```ballerina
@xmldata:Namespace {
    uri: "http://namespace2.com"
}
type Root record {

}
```

Reply:
ATM, no and we don't support default NS as well, we decided to include default NS support with upcoming releases, as this improvement is urgent to be shipped.

Reply:
@AzeemMuzammil Let's create github issues for the above scenario, in addtion to the improvement points we discussed during the our last review.

Reply:
https://github.com/ballerina-platform/ballerina-lang/issues/41690